solutions/account-infrastructure-base/variables.tf

```diff
@@ -19,6 +19,10 @@ variable "prefix" {
   type        = string
   description = "An optional prefix to append to all resources created by this solution."
```

Comment:
Suggest to mention that if this start with an upper case letter, the code is going to convert it to lower case for COS bucket naming if `provision_atracker_cos` is true